frontend/locale/en.json

```diff
@@ -9,28 +9,68 @@
   "404.message": "Sorry the page came back with a 404 error we can't find what you are looking for.",
   "footer.credit": "Created with {link}",
   "footer.icon_credit": "Icons made by {link} ({ccLink})",
-  "nav.brand": "Open Eats",
-  "nav.news": "News",
-  "nav.recipes": "Browse Recipes",
-  "nav.login.title": "Login",
-  "nav.accountmenu.title": "My Account",
-  "nav.accountmenu.create_recipe": "Create Recipe",
-  "nav.accountmenu.grocery_list": "Grocery List",
-  "nav.accountmenu.logout": "Logout",
   "browse.no_results": "Sorry, there are no results for your search.",
   "filter.filter_x": "Filter {title}",
   "filter.clear_filter": "Clear filter",
   "filter.x_stars": "{rating, number} stars",
   "pagination.newer": "Newer",
   "pagination.older": "Older",
-  "searchbar.label": "Search for Recipes",
+  "searchbar.label": "Search for recipes",
+  "searchbar.mobile.label": "Search",
   "searchbar.placeholder": "Enter a title, tag or ingredient.",
   "file_widget.help_block": "Please upload a picture of the finished recipe!",
   "select_widget.header": "Please select a {label}",
   "widget_error.title": "Form error!",
   "widget_error.message": "Please confirm that all the data is there.",
+  "nav.create_recipe": "Create recipe",
+  "nav.import": "Import recipe",
+  "nav.grocery_list": "Grocery list",
+  "nav.login.title": "Login",
+  "nav.accountmenu.title": "My account",
+  "nav.accountmenu.logout": "Logout",
+  "nav.accountmenu.admin": "My account",
+  "nav.brand": "OpenEats",
+  "nav.news": "News",
+  "nav.recipes": "Browse recipes",
+  "grocery_list.my_lists": "My Lists",
+  "grocery_list.footer": "Double Click to edit an item.",
+  "list.new-input-placeholder": "What do you need to buy?",
```

Comment:
because of a not really appropriate german translation (can be translated back into something like "What do you need?"), i thought a little bit more about **"list.new-input-placeholder"**: why has everything to be bought? - i know it is a grocery list, but what if you have a little garden or something like that...?
what do you think about a more general wording like _"What is still left?"_, _"What lacks on the list?"_, _"Is something still missing?"_ or a mixture of them?

Reply:
maybe:
- What else do you need?
- What do you need?

Reply:
well... although the translations (at least the english & german one) now match correct, it is a ungainly solution and from my point of view it got worse. :-(
maybe i dislike this wording because "_What do you need?_" (= "_Was brauchen/brauchst Sie/Du?_") is used by drug-dealers to start a transaction... so our grocery list sounds like a very polite drug-dealer. `*lol*`

i go with my suggestions: (now prioritized)
```
What is (still) missing (on the list)?	=	Was fehlt (noch) (auf der Liste)?
What is (still) left (on the list)?	=	Was fehlt (noch) (auf der Liste)?
What is (still) lacking on the list?	=	Was fehlt (noch) auf der Liste?
What lacks on the list?			=	Was fehlt auf der Liste?
What else do you need?			=	Was benötigst du noch?
Is something still missing?		=	Fehlt noch etwas?
```

Reply:
My favorite is `What else do you need?` and just dropping the `to buy`.

Reply:
you are the boss... i'm ok with it.